clients/privacy-center/cypress/e2e/consent-banner-gpp.cy.ts

```diff
@@ -554,4 +554,202 @@ describe("Fides-js GPP extension", () => {
       });
     });
   });
+
+  describe("with TCF disabled and GPP enabled, with experience region outside of GPP", () => {
+    beforeEach(() => {
+      cy.fixture("consent/experience_gpp.json").then((payload) => {
+        stubConfig({
+          options: {
+            isOverlayEnabled: true,
+            tcfEnabled: false,
+          },
+          // override with a region that is outside GPP jurisdictions
+          experience: { ...payload.items[0], region: "us_nc" },
```

Comment:
This part of the setup is unique to this scenario, but the rest is all shared right